projects/v3/src/app/services/utils.service.ts

```diff
@@ -719,4 +719,23 @@ export class UtilsService {
     this.broadcastEvent('support-email-checked', false);
     return false;
   }
+
+  getSupportEmail() {
+    const expId = this.storageService.getUser().experienceId;
+    const programList = this.storageService.get('programs');
+    if (!expId || !programList || programList.length < 1) {
```

Comment:
@sasangachathumal the `programList.length < 1` implies it could possibly go lower than 0 while length can never become negative value.
it's more recommended to be explicit to keep code readability 😉